cts-java/src/test/java/org/ga4gh/cts/api/variants/VariantsMethodsEndpointAliveIT.java

```diff
@@ -0,0 +1,71 @@
+package org.ga4gh.cts.api.variants;
+
+import org.ga4gh.ctk.CtkLogs;
+import org.ga4gh.ctk.transport.GAWrapperException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.methods.SearchCallSetsRequest;
+import org.ga4gh.methods.SearchVariantsRequest;
+import org.ga4gh.methods.SearchVariantsResponse;
+import org.ga4gh.models.CallSet;
+import org.ga4gh.models.Variant;
+import org.ga4gh.models.VariantSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Verify basic operations on variants, variantsets, and callsets.
+ */
+@Category(VariantsTests.class)
+public class VariantsMethodsEndpointAliveIT implements CtkLogs {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Check that searching {@link Variant}s with a nonexistent reference name returns
+     * an empty list of {@link Variant}s.
+     *
+     * @throws Exception if something goes wrong
+     */
+    @Test
+    public void testSearchVariantsForNonexistentReferenceNameReturnsEmptyList() throws Exception {
+        final String variantSetId = Utils.getVariantSetId(client);
+
+        final SearchVariantsRequest request =
+                SearchVariantsRequest.newBuilder()
+                                     .setReferenceName(Utils.randomName())
+                                     .setVariantSetId(variantSetId)
+                                     .setStart(0L)
+                                     .setEnd(1L)
+                                     .build();
+
+        final SearchVariantsResponse resp = client.variants.searchVariants(request);
+
+        assertThat(resp.getVariants()).isEmpty();
+    }
+
+    /**
+     * Check that searching {@link CallSet}s with the ID of a nonexistent {@link VariantSet} fails.
+     *
+     * @throws Exception if something goes wrong
+     */
+    @SuppressWarnings("ThrowableResultOfMethodCallIgnored")
+    @Test
+    public void testSearchCallSetsForNonexistentVariantSetFails() throws Exception {
+        final SearchCallSetsRequest scsr =
+                SearchCallSetsRequest.newBuilder()
+                                     .setVariantSetId(Utils.randomId())
+                                     .build();
+
+        final GAWrapperException gae =
+                Utils.catchGAWrapperException(() -> client.variants.searchCallSets(scsr));
+
+        assertThat(gae.getHttpStatusCode()).isEqualTo(HttpURLConnection.HTTP_NOT_FOUND);
```

Comment:
It can't be null.  Check the implementation of `catchGAWrapperException`: it assumes (asserts) that the enclosed expression will throw an exception, and if it does not, it calls `fail()`, which causes control to transfer out of the entire test.